docs/testing.md

```diff
@@ -16,16 +16,22 @@ Included with forklift is a small live test suite.  The current tests are:
   * fb-destroy-organization.bats - Cleans up after the content tests
   * fb-finish.bats - Collects logs pertinent to the bats run
 
-To execute the bats framework:
+## Executing the Bats pipeline
 
- * Using vagrant (after configuring vagrant according to this document):
+ ### Using Vagrant
 
+ (after configuring vagrant according to this document):
+
+```
     vagrant up centos7-pipeline-bats
```

Comment:
If you add backticks, you should remove indenting
```suggestion
vagrant up centos7-pipeline-bats
```

Also wondering if this still makes sense since no such box exists. Perhaps this part should be dropped.

Reply:
If the box doesn't exist, no reason to leave this in there.  Dropping.